### ISSUE TITLE
Workers added via packages dont get an invite?

### ISSUE THREAD

**Issue by orenmazor** [jstanden/cerb#1224]
Hey @jstanden not sure if I'm reading this right, but empirically it looks like workers added via package don't get an invite email. Is there a way I can trigger that programmatically?

**Comment by jstanden** [comment]
@orenmazor Yeah, workers created from a package don't receive an invite. However, you could just send them an email with a link to the `/login/recover?email=worker@example.com` page on your site. That will verify their email address and let them set up a password. That endpoint is hardened once the worker has a password, secret questions, and/or (even better) an MFA device.

We can add an option to packages for sending a proper invite, and move that code from the UI action into the lower-level records API. The UI would just set the same option (or always trigger on the lack of a password being set for new users).

Perhaps a better optional flag would be to ignore sending an invite. We'd want something like that in the packages we use for setting up test environments or doing migrations/imports.

**Comment by orenmazor** [comment]
hey @jstanden 

that workaround makes sense to me (and is what I was thinking we'd be doing, one way or another). 

looking at the code, it seemed that you folks trigger an invite email whenever a passwordless user is saved, right? I think that might make that most sense if it were generalized
